app/setup.py

```diff
@@ -417,7 +417,7 @@ def setup_secure_cookies(application):
 
 
 def setup_babel(application):
-    application.babel = Babel(application)
+    application.babel = Babel(application, default_locale='en_GB')
```

Comment:
This `default_locale` is no longer needed.

Reply:
Removed